solver/src/settlement_submission/flashbots_api.rs

```diff
@@ -8,6 +10,20 @@ pub struct FlashbotsApi {
     client: Client,
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+struct Eip1559 {
+    max_fee_per_gas: String,
+    max_priority_fee_per_gas: String,
```

Comment:
Also agree with @vkgnosis:
```suggestion
    #[serde(with = "model::u256_decimal")]
    max_fee_per_gas: U256,
    #[serde(with = "model::u256_decimal")]
    max_priority_fee_per_gas: U256,
```

Reply:
Fixed.